libnetwork/sandbox_dns_unix.go

```diff
@@ -295,27 +295,6 @@ func (sb *Sandbox) setupDNS() error {
 	return rc.WriteFile(sb.config.resolvConfPath, sb.config.resolvConfHashFile, filePerm)
 }
 
-// Called when an endpoint has joined the sandbox.
-func (sb *Sandbox) updateDNS(ipv6Enabled bool) error {
```

Comment:
- The windows implementation of this method can also be deleted. See https://github.com/moby/moby/blob/c6aaabc9fc8201496379ab12bf4c9386000cb21d/libnetwork/sandbox_dns_windows.go#L25-L27
- I think the following TODO comment can be updated / removed: https://github.com/moby/moby/blob/c6aaabc9fc8201496379ab12bf4c9386000cb21d/libnetwork/sandbox_dns_unix.go#L366-L371

Reply:
Thank you - done.